packages/account/src/Components/reset-trading-password-modal/reset-trading-password-modal.tsx

```diff
@@ -44,19 +52,24 @@ const ResetTradingPassword = ({
 
     const validateReset = (values: FormikValues) => {
         const errors: FormikErrors<FormikValues> = {};
+        const new_max_length = platform === CFD_PLATFORMS.MT5 ? 16 : 25;
```

Comment:
please avoid using word `new` since it will not be new anymore very soon
```suggestion
        const max_length = platform === CFD_PLATFORMS.MT5 ? 16 : 25;
```